aspnetcore/mvc/views/tag-helpers/th-components.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Tag Helper Components in ASP.NET Core
+author: scottaddie
+description: Learn what Tag Helper Components are and how to use them in ASP.NET Core.
+monikerRange: '>= aspnetcore-2.0'
+ms.author: scaddie
+ms.date: 09/17/2018
+uid: mvc/views/tag-helpers/th-components
+---
+# Tag Helper Components in ASP.NET Core
+
+By [Scott Addie](https://twitter.com/Scott_Addie) and [Fiyaz Bin Hasan](https://github.com/fiyazbinhasan)
+
+A Tag Helper Component is a Tag Helper that allows you to conditionally modify or add HTML elements from server-side code. This feature is available in ASP.NET Core 2.0 or later.
+
+ASP.NET Core includes two built-in Tag Helper Components: `head` and `body`. They're located in the `Microsoft.AspNetCore.Mvc.Razor.TagHelpers` namespace and can be used in both MVC and Razor Pages. The Tag Helper Components are registered with the app in *_ViewImports.cshtml* like any other Tag Helper:
```

Comment:
They aren't. They're built-in and don't require the Mvc.Razor.TagHelpers `@addTagHelper` bits.